src/browser/reset.js

```diff
@@ -57,9 +60,9 @@ export async function reset(opts = {}) {
 		currentRtl = opts.rtl;
 	}
 
-	if (opts.lang !== currentLang) {
-		document.documentElement.setAttribute('lang', opts.lang);
-		currentLang = opts.lang;
+	opts.lang ??= '';
+	if (documentLocaleSettings.lamguage !== opts.lang) {
```

Comment:
I assume this should be la**n**guage and this is working right now because this is always true?